.github/workflows/hil_sample_esp-idf.yml

```diff
@@ -1,6 +1,23 @@
 name: ESP-IDF Hardware-in-the-Loop Sample Tests
 
 on:
+  workflow_dispatch:
+    inputs:
+      hil_board:
+        required: true
+        type: string
+      idf_target:
+        required: true
+        type: string
+      api-url:
+        required: true
+        type: string
+      api-key-id:
+        required: true
+        type: string
+      coap_gateway_url:
+        required: true
+        type: string
```

Comment:
mirrored the no defaults / no description in each of these for `workflow_call` -- happy to add them if helpful! Imagine that folks invoking these are going to be quite familiar with the parameters without descriptions / defaults tho